runtime/src/main/java/org/corfudb/protocols/service/CorfuProtocolMessage.java

```diff
@@ -45,6 +42,18 @@ private CorfuProtocolMessage() {}
         IGNORE
     }
 
+    /**
+     * Returns a ProtocolVersionMsg with empty capability_vector and static
+     * corfu_source_code_version.
+     *
+     * @return A ProtocolVersionMsg with empty capability_vector.
+     */
+    public static ProtocolVersionMsg getDefaultProtocolVersionMsg() {
+        return ProtocolVersionMsg.newBuilder()
+                .setCorfuSourceCodeVersion(GitRepositoryState.getCorfuSourceCodeVersion())
```

Comment:
Would it be better to initialize the compatibility vector as well? It seems like right now we are using an empty list to denote the default capabilities instead of a vector of all zeros.

Reply:
From our discussion this part would be handled in a separate PR for capability vector implementation.